GLFWViewer/GLFWViewer.cpp

```diff
@@ -272,8 +278,12 @@ bool GLFWViewer::Draw() {
 	for (int geometryIndex = 0; geometryIndex < m_VAOS.size(); geometryIndex++) {
 		glBindVertexArray(m_VAOS[geometryIndex]);
 		// Bind the Model Matrix corrosponding to the current Geometry
-		SetGeometryModelMatrix(m_scene->m_geometries[geometryIndex]->m_modelMatrix);
-		glDrawArrays(GL_TRIANGLES, 0, m_scene->m_geometries[geometryIndex]->m_triangleIndices.size());
+		std::shared_ptr<Geometry> geometryPtr = m_scene->m_geometries[geometryIndex];
+		if (geometryPtr->GetGeomtryType() == GeometryType::TRIANGLEMESH) {
```

Comment:
so right now you draw only if the geometry is a TRIANGLEMESH? how would we handle other geometries?